electronics/platformio.ini

```diff
@@ -12,19 +12,11 @@
 platform = ststm32
 board = nucleo_h743zi
 framework = mbed
-build_flags = -std=c++17 -fno-strict-aliasing
-
-; debug_tool = stlink
-
-; Use our fixed board definition for debug and upload, since the stock one that comes with
```

Comment:
Please bring back the comments, they were there for a reason

Reply:
oh, whoops. I deleted that by accident when I was trying to fix something by adding flags